appengine-python-flask/main.py

```diff
@@ -62,8 +64,8 @@
     extensions=['jinja2.ext.autoescape'],
     autoescape=True)
 
-TOPIC = "shout-tasks"
-SUBSCRIPTION = "shout-tasks-workers"
+TOPIC = "shout-requests"
+SUBSCRIPTION = "shout-request-workers"
```

Comment:
If they run > 1 instances of this app (or different versions of the same app) it looks like they will share the same topic and subscription. This doesn't cause errors, but is probably not what you want. You could use the app id + version to distinguish.